scripts/tts_dataset_files/cmudict-0.7b_nv22.01

```diff
@@ -133908,3 +133872,241 @@ ZYWICKI  Z IH0 W IH1 K IY0
+BOT  B AA1 T
+BREAKPOINT  B R EY1 K P OY2 N T
+BRONCHOSCOPY  B R AA0 N K AA1 S K AH0 P IY0
+CALENDARING  K AE1 L AH0 N D R IH0 NG
+CALTECH  K AH1 L T EH1 K
+CDAC  S IY1 D AE1 K
+CES  S IY1 IY1 EH1 S
+CHATBOT  CH AE1 T B AA2 T
+COMPOSABLE  K AH0 M P OW1 Z AH0 B AH0 L
+COMPUTATIONALLY  K AA2 M P Y UW0 T EY1 SH AH0 N AH0 L IY0
+CONFIG  K AH1 N F IH2 G
+COREWARE  K AO1 R W EH2 R
+COVID  K OW1 V IH0 D
+CPU  S IY1 P IY1 Y UW1
+CROSSHAIR  K R AO1 S HH EH1 R
+CRYO  K R AY1 OW1
+CSP  S IY1 EH1 S P IY1
+CTR  S IH1 T IY1 AA1 R
+CUDA  K UW1 D AH0
+CUDNN  K UW1 D IY1 EH2 N EH2 N
+CYBERCRIME  S AY1 B ER0 K R AY1 M
+DATACENTER  D EY1 T AH0 S EH2 N T ER0
+DDOS  D IY2 D AO1 S
+DEAFHEAVEN  D EH1 F HH EH1 V AH0 N
+DGX  D IY1 JH IY1 EH1 K S
+DIRECTX  D IH0 R EH1 K T EH1 K S
+DJ  D IY1 JH EY1
+DLSS  D IY1 EH1 L EH1 S EH1 S
+DNM  D IY1 EH2 N EH1 M
+DOCA  D OW1 K AH0
+DPU  D IY1 P IY1 Y UW1
+DS  D IY1 EH1 S
+DYSTOPIA  D IH1 S T OW1 P IY0 AH0
+DYSTOPIAN  D IH1 S T OW1 P IY0 AH0 N
+EGX  IY1 JH IY1 EH1 K S
+ENCODER  EH0 N K OW1 D ER0
+ETL  IY1 T IY1 EH1 L
+EXA  EH1 K S AH0
+F1  EH1 F W AH1 N
+FILENAME  F AY1 L N EY2 M
+FLOWTRON  F L OW2 T R AA1 N
+FORTNITE  F AO1 R T N AY2 T
+FP  EH1 F P IY1
+FPS  EH1 F P IY1 EH1 S
+G6  JH IY1 S IH1 K S
+G6X  JH IY1 S IH1 K S EH1 K S
+GCP  JH IY1 S IY1 P IY1
+GE  JH IY0 IY1
+GEFORCE  JH IY1 F AO1 R S
+GI  JH IY1 EY1
+GIGAHERTZ  G IH1 G AH0 HH EH2 R T S
+GMAIL  JH IY1 M EY1 L
+GPU  JH IY1 P IY1 Y UW1
+GPUS  JH IY1 P IY1 Y UW1 Z
+GSK  JH IY1 EH1 S K EY1
+GT  JH IY1 T IY1
+GTC  JH IY1 T IY1 S IY1
+GTX  JH IY1 T IY1 EH1 K S
+HB  EY1 CH B IY1
+HD  EY1 CH D IY1
+HDMI  EY2 CH D IY1 EH1 M EY1
+HP  EY1 CH P IY1
+HPC  EY1 CH P IY1 S IY1
+IMAC  AY0 M AE1 K
+INITIALISM  IH2 N IH1 SH AH0 L IH2 Z AH0 M
+INT  IH1 N T
+INVERTIBLE  IH2 N V ER1 T AH0 B AH0 L
+IOT  AY1 OW1 T IY1
+IPSEC  AY2 P IY2 S EH1 K
+JEN-HSUN  JH EH1 N S UW0 N
+KILOHERTZ  K IH1 L AH0 HH EH2 R T S
+KIWIBOT  K IY1 W IY0 B AA2 T
+LAUNCHPAD  L AO1 N CH P AE2 D
+LSI  EH2 L EH2 S AY1
+METAVERSE  M EH1 T AH0 V ER2 S
+METAVERSES  M EH1 T AH0 V ER2 S IH0 Z
+MICROSERVICE  M AY2 K R OW0 S ER1 V IH0 S
+MINECRAFT  M AY1 N K R AE2 F T
+ML  EH2 M EH1 L
+MLOPS  EH2 M EH2 L AA2 P S
+MLPERF  EH2 M EH2 L P ER1 F
+MSI  EH1 M EH1 S AY1
+NADELLA  N AA0 D EH1 L AH0
+NANOPORE  N AA1 N OW0 P AO0 R
+NGC  EH1 N JH IY1 S IY1
+NGX  EH1 N JH IY1 EH1 K S
+NHS  EH1 N EY1 CH EH1 S
+NIH  EH1 N AY1 EY1 CH
+NVIDIA  EH0 N V IY1 D IY0 AH0
+NVIDIA'S  EH0 N V IY1 D IY0 AH0 Z
+NVLINK  EH1 N V IY1 L IH1 NG K
+NVME  EH1 N V IY1 EH1 M YI1
+NVSWITCH  EH1 N V IH1 S W IH1 CH
+NVSWITCHES  EH1 N V IH1 S W IH1 CH IH0 Z
+NVTABULAR  EH1 N V IH1 T AE1 B Y AH0 L AA1 R
+OEM  OW1 IY1 EH1 M
+OFFLOAD  AO1 F L OW2 D
+OMNIVERSE  AA1 M N IY0 V ER1 S
+OPENSTACK  OW1 P AH0 N S T AH1 K
+OPTIMIZED  AA1 P T IH0 M AY2 Z D
+OPTIMIZING  AA1 P T IH0 M AY2 Z IH0 NG
+PARABRICK  P EH1 R AH0 B R IH0 K
+PCB  P IY1 S IY1 B IY1
+PCI  P IY1 S IY1 EY1
+PCIE  P IY1 S IY1 EY1 IY1
+PERFORMANT  P ER0 F AO1 R M AH0 N T
+PETAFLOP P EH1 T AH0 F L AA2 P
```

Comment:
This line is formatted incorrectly. Should be a double space after PETAFLOP not a single space

Reply:
Thanks, good catch (x3).